app/models/service_template.rb

```diff
@@ -3,6 +3,7 @@ class ServiceTemplate < ActiveRecord::Base
   include ServiceMixin
   include OwnershipMixin
   include NewWithTypeStiMixin
+  include ServiceTemplateFilter
```

Comment:
This should be `include_concern 'Filter'` and then you will need to move service_template_filter.rb to `app/models/service_template/filter.rb'